### PR TITLE
story/TYTA-41/setup-project-model

### PR DESCRIPTION
**Pull request by BrystoQ**
##Description
Add a model for project

### CODE REVIEW

File: src/models/projectModel.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          default: 'user',
+        },
+      },
+    ],
+    team: {
+      id: {
+        type: Schema.Types.ObjectId,
+        ref: 'Project',
+      },
+    },
```
Comment (remirubis) [inline src/models/projectModel.js]: I'm not sure why you create a child object with only id, instead you can do this:
```suggestion
    team: {
      type: Schema.Types.ObjectId,
      ref: 'Project',
    },
```

**Review by remirubis — CHANGES_REQUESTED** [review]
Just a simple question about user team implementation in this model

File: src/models/projectModel.js (hunk 1 above)
Comment (BrystoQ) [reply src/models/projectModel.js]: My bad, didn't know that was possible simply like this

**Review by remirubis — APPROVED** [review]
LGTM! 🚀

**Review by FabienHP — APPROVED** [review]

**Pull request merged**